share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -52,12 +52,22 @@ default:
       - runner_system_failure
       - stuck_or_timeout_failure
 
+.generate-aarch64:
+  extends: [ ".generate" ]
+  tags: ["spack", "public", "medium", "aarch64"]
```

Comment:
Whoops, I forgot an `aws` in here.  Our new policy is run all pipeline generation jobs on runners under our control.  Fix coming shortly.